frontend/yarn.lock

```diff
@@ -2786,7 +2786,7 @@ __metadata:
   languageName: unknown
   linkType: soft
 
-"@opetushallitus/kieli-ja-kaantajatutkinnot.shared@workspace:packages/shared, shared@npm:@opetushallitus/kieli-ja-kaantajatutkinnot.shared@1.5.8":
+"@opetushallitus/kieli-ja-kaantajatutkinnot.shared@workspace:packages/shared":
```

Comment:
Dev-haaran yarn.lock tais olla väärässä tilassa kun se muuttuu tän pullarin yhteydessä?

Reply:
En oo ihan varma miksi mutta pipeline hajosi jostain syystä tässä haarassa. Mergeen näin kun näyttää toimivan ja korjailen deviä sitten tarvittaessa.